packages/bs-neo3/src/BSNeo3.ts

```diff
@@ -121,9 +123,21 @@ export class BSNeo3<BSCustomName extends string = string>
   }
 
   async decryptKey(encryptedKey: string, password: string): Promise<Account> {
-    const wif = await wallet.decrypt(encryptedKey, password)
-    const { address } = new wallet.Account(wif)
-    return { address, wif }
+    let key: string
+    try {
+      const { NativeModules } = require('react-native')
+
+      if (!NativeModules.BsReactNativeDecrypt) {
+        throw new Error('React native decrypt module is not installed')
+      }
+
+      key = await NativeModules.BsReactNativeDecrypt.decryptNeo3(encryptedKey, password)
```

Comment:
I didn't undertand at first that you were also catching a possible error on line 134. So I thought "Why he is throwing and then catching the error instead of putting what is on the catch inside the IF.

Maybe there is a more explicit way to do this, but whatever.